electroncash/wallet.py

```diff
@@ -3522,8 +3523,14 @@ def remove_payment_request(self, addr, config, clear_address_label_if_no_tx=True
             self.save_payment_requests()
         return True
 
-    def get_sorted_requests(self, config):
+    def get_sorted_requests(self, config, *, filter_asset: str = None):
```

Comment:
Technically the annotation should be Optional[str]: 

```suggestion
    def get_sorted_requests(self, config, *, filter_asset: Optional[str] = None):
```